docs/docs/sourcing-from-graphcms.md

```diff
@@ -0,0 +1,162 @@
+        // The top level query type, can be anything you want!
+        typeName: "GCMS",
+        // The field you'll query against, can also be anything you want.
+        fieldName: "gcms",
+        // Your API endpoint, available from the dashboard and settings window.
+        // You can use this endpoint that features US mountains for now.
+        url: "https://api-euwest.graphcms.com/v1/cjm7tab4c04ro019omujh708u/master",
+    },
+},
+```
+
+If everything works correctly, you should now have your GraphCMS data added to the Gatsby source API!
+
+### Querying for Content
+
+From the root of your project, run the development environment with `yarn run development`. Once the server has started and is error free, you should be able to open the following URL in your browser:
+
+http://localhost:8000/___graphql
+
+This will show you a "Graphiql" playground where you can test your new content API.
+
+Try running this query:
+
+```js
+query {
+    gcms {
+        mountains {
+            title
+            elevation
+            image {
+                url
+            }
+        }
+    }
+}
+```
+
+Again, if everything is working properly, you should see a successful response in the shape of:
+
+```json
+{
+  "data": {
+    "gcms": {
+      "mountains": [
+        {
+          "title": "Denali",
+          "elevation": 6190,
+          "image": {
+            "url": "https://media.graphcms.com//J0rGKdjuSwCk7QrFxVDQ"
+          }
+        },
+        {
+          "title": "Mount Elbert",
+          "elevation": 4401,
+          "image": {
+            "url": "https://media.graphcms.com//JNonajrqT6SOyUKgC4L2"
+          }
+        }
+        // ...more results
+      ]
+    }
+  }
+}
+```
+
+### Getting Content on the Page
+
+For the purpose of this tutorial I've removed all the layout, SEO, link or other special components that comprise a page in the Gatsby starter. Open the index file located at `src/pages/index.js` and replace the content with this code:
+
+```js
+import React from "react"
+import { StaticQuery } from "gatsby"
+
+const IndexPage = () => (
+  <StaticQuery
+    query={graphql`
+      query {
+        gcms {
+          mountains {
+            title
+            elevation
+          }
+        }
+      }
+    `}
+    render={data => (
+      <div>
+        <h1>USA Mountains</h1>
+        <ul>
+          {data.gcms.mountains.map(mountain => {
+            const { title, elevation } = mountain
+            return (
+              <li>
+                <strong>{title}:</strong>
+                {elevation}
+              </li>
+            )
+          })}
+        </ul>
+      </div>
+    )}
+  />
+)
+
+export default IndexPage
+```
+
+With this code, you have:
+
+1. Added the "StaticQuery" component to a page that allows you to fetch content from the GraphQL API.
+2. Fetched some simplified data from the Mountain API, namely the title and elevation.
+3. Rendered the list in the StaticQuery's RenderProp called "render".
```

Comment:
```suggestion
3. Rendered the list in the `StaticQuery`'s `render` prop.
```

Reply:
I'm ok with this. Would you not bother telling them that render uses the "render prop" pattern?

Reply:
Unless you wanted to link to a definition of what "render props" are, I wouldn't worry about it here.

Reply:
👍